### PR TITLE
Add AuthDisplayer for redirect unauthiorized users

### PR DESCRIPTION
**Pull request by okaufmann**

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Please rebase this. :)

**Comment by okaufmann** [comment]
After my disput with the rebase file, I finally clean up the mess. Sorry for that. Hope it fits and helps now :sweat_smile:

File: src/Displayers/AuthDisplayer.php [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+    /**
+     * Create a new view displayer instance.
+     *
+     * @param \GrahamCampbell\Exceptions\ExceptionInfo $info
+     * @param \Illuminate\Contracts\View\Factory       $factory
+     *
+     * @return void
+     */
+    public function __construct(ExceptionInfo $info, Factory $factory)
```
Comment (GrahamCampbell) [inline src/Displayers/AuthDisplayer.php]: You can delete this constructor and all the properties

**Comment by GrahamCampbell** [comment]
Needs tests. :)

**Comment by GrahamCampbell** [comment]
Please don't make any modifications to the traits or other code. Only add a displayer and tests.

**Comment by okaufmann** [comment]
Forgot the open PR and need a fast fix for the validation behaviour. My bad.

**Pull request closed without merging**